developer-docs-site/docs/guides/move-guides/mint-nft-cli.md

```diff
@@ -3,76 +3,188 @@ title: "Mint an NFT with Aptos CLI"
+  ```shell
+  aptos init --profile nft-receiver
+  ```
+
+3. When prompted for a network, select `testnet` by entering it and hitting return:
+  ```shell
+  Configuring for profile nft-receiver
+  Choose network from [devnet, testnet, mainnet, local, custom | defaults to devnet]
+  testnet
+  ```
+
+4. When prompted for your private key, hit enter to generate a new key:
+  ```shell
+  Enter your private key as a hex literal (0x...) [Current: None | No input: Generate new key (or keep one if present)]
+  ```
+
+5. Receive output resembling:
+  ```shell
+  No key given, generating key...
+  Account a233bf7be2b93f1e532f8ea88c49e0c70a873d082890b6d9685f89b5e40d50c2 does not exist, you will need to create and fund the account through a community faucet e.g. https://aptoslabs.com/testnet-faucet, or by transferring funds from another account
+  
+  ---
+  Aptos CLI is now set up for account a233bf7be2b93f1e532f8ea88c49e0c70a873d082890b6d9685f89b5e40d50c2 as profile nft-receiver!  Run `aptos --help` for more information about commands
+  {
+    "Result": "Success"
+  }
+  ```
+
+6. Note your configuration information can be found in `.aptos/config.yaml` relative to where you ran `aptos init`. Read that file to see each profile's private and public keys, account address, and REST API URL.
+
+### Install wallet and import account
+
+1. Read `.aptos/config.yaml` to see and copy the `nft-receiver` private key.
+
+2. Install the wallet of your choice. We use the [Petra Wallet](../../guides/install-petra-wallet.md) Chrome extension.
+
+3. Open the wallet and select the [Testnet network](https://petra.app/docs/use) in the wallet via *Petra settings > Network > Testnet*.
+
+4. Go to *Petra > Settings > Switch account > Add Account > Import private key*.
+
+5. Paste the `nft-receiver` private key there.
+
+6. Click **Submit** to add the previously created account to the wallet.
+
+7. You are switched into that account automatically.
+
+### Get coins from faucet
+
+1. Go to the *Petra > Settings > Network > Testnet* network if not there already.
+
+2. Connect your wallet to the Aptos faucet at https://aptoslabs.com/testnet-faucet:
+
+  ![Faucet connect](../../../static/img/connect-wallet-faucet.png "Connect faucet to wallet")
+
+3. Select your wallet type:
 
-1. Receive results resembling:
+  ![Wallet select](../../../static/img/select-wallet-faucet.png "Select your wallet for faucet")
+
+4. Approve the connection request:
+
+  ![Faucet approval](../../../static/img/approve-wallet-faucet.png "Approve connecting faucet to wallet")
+
+5. Now when you load your wallet, you will see a **Faucet** button next to **Send**. Click **Faucet** to receive one APT per click to use when minting.
+
+### Mint the NFT
+
+1. Mint the NFT by calling the `mint_nft` function and an existing contract using the Aptos CLI:
+
+  ```shell
+  aptos move run --function-id 8cdf69c8c93fee36ed83f8882908060c1335ed39a827c08dbb506b46237e88fb::minting::mint_nft --profile nft-receiver
```

Comment:
ah so this I think will have to change, because the new tutorial we're asking them to deploy their own contract. so the `8cdf69c8c93fee36ed83f8882908060c1335ed39a827c08dbb506b46237e88fb` part would be different depending on which resource address they deployed the contract to

Reply:
Thanks, @0xchloe . Where do they get this value? From config.yaml? If so, what field? See the internal Workshop #1 page for related questions.

Reply:
I think this will be printed out when they create the resource account

Reply:
I no say prominently:
"Note your configuration information can be found in .aptos/config.yaml relative to where you ran aptos init. Read that file to see each profile's private and public keys, account address, and REST API URL."

So I believe this is addressed.